### PR TITLE
Update to use new combined repo, and structs instead of custom packing

### PR DESCRIPTION
**Pull request by Arachnid**
One of the 'real' tests is presently commented out because of the change in how we handle expirations; since we now only treat records as expired when the signature expiration is reached, we cannot simply resubmit the same records in the test.

The signature in the test case for TXT _ens.matoken.xyz expires July 19, at which point we can fetch a new one and re-enable that test case.

### CODE REVIEW

**Comment by makoto** [comment]
`The signature in the test case for TXT _ens.matoken.xyz expires July 19, at which point we can fetch a new one and re-enable that test case.` so we need to remember to update the test every time it expires? Is there any way to automate to at least auto-generate the test case or something?

**Comment by Arachnid** [comment]
@makoto This is just a one time thing - we need two sets of 'real' data, the second one from after the first one expires. I've set a calendar reminder for myself to fetch the second set of data.

File: src/oracle.ts [hunk 1]
```diff
@@ -63,24 +68,12 @@ export class Oracle {
         if(serialNumberGt(inception, proof.signature.data.inception)) {
             throw new OutdatedDataError(proof);
         }
-        const expired = serialNumberGt(this.now() / 1000, expiration + proof.signature.data.originalTTL);
```
Comment (makoto) [inline src/oracle.ts:66]: Are there any test cases that were passing previously but now fails due to this change?

Comment (Arachnid) [reply src/oracle.ts:66]: Yes, the one that's commented out until we get more data.

**Review by makoto — APPROVED** [review]

**Pull request merged**